jujugui/static/gui/src/app/components/charmbrowser/test-charmbrowser.js

```diff
@@ -43,14 +43,12 @@ describe('Charmbrowser', function() {
         }
       }};
     var changeState = sinon.stub();
-    var addService = sinon.stub();
     var charmstoreSearch = sinon.stub();
     var output = jsTestUtils.shallowRender(
       <juju.components.Charmbrowser
         appState={appState}
         changeState={changeState}
-        charmstoreSearch={charmstoreSearch}
-        addService={addService} />);
```

Comment:
I'm not sure what the impact would be, but based on my other refactor PR (to remove the charmstore dependency from `getIconPath`) I'm guessing changing `addService` would not be trivial. We already have the `getEntity` method in the `EntityDetails` component; I'd rather just pass in `deployService` and bypass `addService` altogether.